### PR TITLE
Merging Tanguy's branch with nidification_main

### PR DESCRIPTION
**Pull request by TanguyAubreton**

### CODE REVIEW

File: Analyse_Birds.qmd
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:490]: here I don't think it is look about but with look ity is "to look at"

File: Analyse_Birds.qmd
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:507]: I think it is "there ARE NOT" and not "isn't"
An i am not sure if it is "there could be" maybe it would be better with "it could" but i am not sure

File: Analyse_Birds.qmd
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:587]: I thonk it is "only presentS (with an s) the cont..." and not with "present it"

File: Analyse_Birds.qmd
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:675]: in term of structure i would put a dot after 572.82 and start a new sentence with "In order"

File: Analyse_Birds.qmd
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:692]: it is grammaticaly correct but i would maybe say : First of all, in situations where a counting model is used, such as when following a Poisson distribution

File: Analyse_Birds.qmd
Comment (anaelleduplessis) [inline Analyse_Birds.qmd:701]: sentences structure to maybe check

**Comment by MarieEtienne** [comment]
PLease do not merge right now, there issues in the master branch which needs to be fixed before

File: Analyse_Birds.qmd
Comment (OceaneGourdin) [inline Analyse_Birds.qmd:460]: Isn't this rather part I. ?

**Review by OceaneGourdin — APPROVED** [review]
The analysis seems good to me. I validate the pull request, but make sure to modify the grammatical errors highlighted by Anaëlle.

**Review by rudyusach — APPROVED** [review]

**Review by VJSTT — APPROVED** [review]
Well done

**Review by anaelleduplessis — APPROVED** [review]

**Pull request merged**
